
Starting trainig
Not found pretrained weights. Continue without any pretrained weights.
Training Data Size : 50400
Validation Data Size : 5600
/opt/conda/lib/python3.7/site-packages/torch/nn/functional.py:3063: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
/opt/conda/lib/python3.7/site-packages/torch/nn/functional.py:2952: UserWarning: nn.functional.upsample is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.functional.upsample is deprecated. Use nn.functional.interpolate instead.")
epoch: 0    0 / 12600
lossD: 3.486025094985962    lossG: 90.35098266601562 batch_time: 10.933068752288818s
L_adv: 3.2245731353759766 L_id: 0.985640287399292 L_attr: 3.6110997200012207 L_rec: 0.3713935613632202
L_l2_eyes: 0.008168606087565422
L_landmarks: 0.003661399707198143
loss_adv_accumulated: 19.66449146270752
itertion :
Traceback (most recent call last):
  File "train.py", line 651, in <module>
    main(args)
  File "train.py", line 527, in main
    train(args, device=device)
  File "train.py", line 394, in train
    loss_adv_accumulated)
  File "train.py", line 256, in train_one_epoch
    res1 = get_faceswap('examples/images/training/source1.png', 'examples/images/training/target1.png', G, id_extractor, device)
  File "/workspace/utils/training/image_processing.py", line 103, in get_faceswap
    id_embedding, src_emb, tgt_emb = id_extractor(source, target)
  File "/opt/conda/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/torch/autograd/grad_mode.py", line 26, in decorate_context
    return func(*args, **kwargs)
  File "/workspace/extractor/SA_idextractor.py", line 60, in forward
    v_id_source = F.normalize(self.f_id(F.interpolate((i_source - 0.5)/0.5, size=112, mode='bilinear')), dim=-1, p=2).to(self.device)
  File "/opt/conda/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/workspace/extractor/arcface_model/iresnet.py", line 142, in forward
    x = self.conv1(x)
  File "/opt/conda/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/torch/nn/modules/conv.py", line 423, in forward
    return self._conv_forward(input, self.weight)
  File "/opt/conda/lib/python3.7/site-packages/torch/nn/modules/conv.py", line 420, in _conv_forward
    self.padding, self.dilation, self.groups)
RuntimeError: Input type (torch.cuda.FloatTensor) and weight type (torch.FloatTensor) should be the same